src/pages/index.js

```diff
@@ -1,7 +1,7 @@
 import Helmet from "react-helmet";
 import React from "react";
 
-import Navigation from "../components/Navigation";
+import DesignSystemNavigation from "../components/DesignSystemNavigation";
```

Comment:
Can you explain why we're calling this the `DesignSystemNavigation` but below isn't `DesignSystemTitle`?

Reply:
Good question. This navigation is specific to the Design System _website_. It's the list of sections on the landing pages. Whereas `Title` is both a part of the Design System itself and used on the Design System _website_